cmd/mtping/main.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	component = "pingsource-mt-adapter"
+	component = "pingsource"
```

Comment:
Thanks @pierDipi for raising this.

The reason to change to `pingsource` is that configuration would be more intuitive for users, since `pingsource-mt-adapter` is now more an internal identifier.

Am I wrong to assume that changing the lease name would produce a re-election only, and maybe if there are multiple controllers, for some brief time 2 of them could be active at the same time?

Reply:
I reverted this change.
I still think users should configure logging ConfigMap by using `pingsource` rather than `pingsource-mt-adapter` which feels less intuitive, but I also think we can discuss the effect of that change in a different issue.

Thanks again for raising the concern.